storage/qcs.go

```diff
@@ -0,0 +1,21 @@
+package storage
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage/badger/transaction"
+)
+
+// QuorumCertificates represents storage for Quorum Certificates.
+// Quorum Certificates are distributed using blocks, each block incorporates a QC for its parent.
+// This specific storage allows to store and query QCs discovered from both consensus participants and followers.
```

Comment:
I think most of our storage components are used by consensus as well as participants. I don't think we need to explicitly mention this here. 
```suggestion
```